grid-compose/internal/utils/mycelium_seed.go

```diff
@@ -0,0 +1,13 @@
+package utils
```

Comment:
it is not a good practice to have a package named `utils`

instead add these methods in the package where it are used, okay to have a separate file for it in the same package.